spring-cloud-release-tools-spring/src/main/java/org/springframework/cloud/release/internal/spring/OptionsAndProperties.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.release.internal.spring;
+
+import org.springframework.cloud.release.internal.ReleaserProperties;
+import org.springframework.cloud.release.internal.options.Options;
+
+class OptionsAndProperties {
```

Comment:
Why don't you use Spring Boot properties for this stuff?

Reply:
Cause those can be passed at runtime from the console and they get updated at runtime depending on what's in the config/releaser.yml file. For its deserialization I'm using boot though.